packages/arco-lib/src/components/Table/Table.tsx

```diff
@@ -215,7 +215,7 @@ export const Table = implementRuntimeComponent({
   } = pagination;
 
   const rowSelectionType = rowSelectionTypeMap[cProps.rowSelectionType];
-  const currentChecked = useRef<(string | number)[]>([]);
+  const currentChecked = useRef<any[]>([]);
```

Comment:
Why use any?

Reply:
Since only the `selectedRowKeys` needed to be recorded before, the type can be determined.     
Now it has been changed to record `selectedRows` and the type of `table row` cannot be determined.  
I have changed the type to `Record<string,unkonwn>[]` and will infer the type when use it